proto/v2/observation.proto

```diff
@@ -73,6 +73,10 @@ message FacetFilter {
   // When specified, only observations with provenance in these domain are
   // returned.
   repeated string domains = 4;
+  // key is variable dcid and value is facet id to filter for. For all variables
+  // with a facet id to filter for, only observations with that facet id are
+  // returned.
+  map<string, string> facet_id = 5;
```

Comment:
instead of adding a map here, how about change line 98 to use repeated filter? Then in facet filter we add a repeated string of stat var dcid.

Reply:
done